lib/records/RecConfigParse.cc

```diff
@@ -152,7 +152,7 @@ RecConfigFileParse(const char *path, RecConfigEntryCallback handler, bool inc_ve
   ink_hash_table_destroy(g_rec_config_contents_ht);
   g_rec_config_contents_ht = ink_hash_table_create(InkHashTableKeyType_String);
 
-  line_tok.Initialize(fbuf, SHARE_TOKS);
+  line_tok.Initialize(fbuf, SHARE_TOKS | ALLOW_EMPTY_TOKS);
```

Comment:
The description of this PR does not match with the `ALLOW_EMPTY_TOKS` option, but it seems like it would correspond to the `ALLOW_SPACES` option.

Can you give an example of what is going wrong with the parsing here? Can we use quoted strings for values in which white-space is important?

Reply:
My title may be off. it should probably be "Preserve empty lines when reading/writing records.config"

This is to address something like this 

```
CONFIG proxy.config.exec_thread.affinity INT 1

##############################################################################
```

getting written back out as

```
CONFIG proxy.config.exec_thread.affinity INT 1
##############################################################################
```

(dropping the empty line)

It's minor annoyance, but it's something I encounter with our configuration management software writing it out in one format, only to have ATS rewrite it differently.

Reply:
Ahh, now I see. Yeah this makes sense. Would you mind changing the commit message to say `Preserve blank lines ...` ?